api/v1beta1/helmrepository_types.go

```diff
@@ -72,6 +72,12 @@ type HelmRepositorySpec struct {
 	// AccessFrom defines an Access Control List for allowing cross-namespace references to this object.
 	// +optional
 	AccessFrom *acl.AccessFrom `json:"accessFrom,omitempty"`
+
+	// Type of the HelmRepository.
+	// When this field is set to  "OCI", the URL field value must be prefixed with "oci://".
+	// +kubebuilder:default:="Default"
```

Comment:
Let’s remove this for now, adding the default in etcd will prevent people from rolling back, as all CRs will not be compatible with older versions of the CRD.

Reply:
addressed in 4c0b0779c94eb718e93b8c979f9e65a12e48df9e